x-pack/plugins/observability/public/pages/alert_details/types.ts

```diff
@@ -5,18 +5,14 @@
  * 2.0.
  */
 
+import { TopAlert } from '../alerts/containers/alerts_page/types';
```

Comment:
I saw this `TopAlert` type previously but I am not sure why this is named like this, Any idea why the type name is `TopAlert` instead of `Alert`?

Reply:
Good question, I was wandering the same. Maybe @simianhacker or @fkanout  knows.

Reply:
🤷‍♂️

Reply:
Same here 🤷‍♂️